mgmtfn/mesosplugin/cniapi/cniapi.go

```diff
@@ -0,0 +1,111 @@
+/***
+Copyright 2016 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// CNI API definitions shared between the Mesos plugin and netplugin
+
+package cniapi
+
+// netplugin socket dir
+const PluginPath = "/run/contiv"
+
+// netplugin socket file
+const ContivMesosSocket = "/run/contiv/contiv-mesos.sock"
+
+// MesosNwIntfAdd: endpoint handling network interface add
+const MesosNwIntfAdd = "/MesosNwIntfAdd"
+
+// MesosNwIntfDel: endpoint handling network interface delete
+const MesosNwIntfDel = "/MesosNwIntfDel"
+
+// network labels from Mesos
+type NetworkLabel struct {
+	Key   string `json:"key,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+// netplugin labels
+type NetpluginLabel struct {
+	TenantName   string `json:"tenant-name,omitempty"`
+	NetworkName  string `json:"network-name,omitempty"`
+	NetworkGroup string `json:"network-group,omitempty"`
+}
```

Comment:
sure, I will change it. So this would be appear in the scheduler (Marathon) as 
 "ipAddress": {
     "networkName": "contiv",
     "labels": {
            io.contiv.tenant=XXX
            io.contiv.network=YYY
            io.contiv.net-group=ZZZ
     }
  }